include/glow/Runtime/RuntimeTypes.h

```diff
@@ -171,6 +171,28 @@ struct HostConfig {
   size_t executorThreads{3};
 };
 
+/// This is struct is for user defined partition.
+struct PartitionConfig {
+  /// The name of the functiton to be partitioned.
+  std::string funcName;
+  /// The number of user defined partitions. The partition ids are [0,
+  /// numOfPartitions - 1];
+  size_t numOfPartitions;
+  /// The backend for each partition. backendNames.size() == numOfPartitions.
+  std::vector<std::string> backendNames;
+  /// The name for each partition. partitionNames.size() == numOfPartitions.
+  std::vector<std::string> partitionNames;
+  /// The mapping between nodes and Partition ids. Assume there are n nodes and
```

Comment:
Is this a map between node's name and unsigned partition number? It wasn't clear to me initially that name is used here. Also, need to clarify that Glow mangles names, meaning that Glow's node name is never the same as name from Caffe2 protobuf.